content/blog/2018-08-23-interview-with-presslabs.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Interview with Presslabs
+created_at: 2018-08-23
+kind: article
+author_name: Brian Brazil
+---
+
+*Continuing our series of interviews with users of Prometheus, Mile Rosu
+from Presslabs talks about their monitoring journey.*
+
+## Can you tell us about yourself and what Presslabs does?
+
+[Presslabs](https://www.presslabs.com/) is a high-performance managed WordPress
+hosting platform targeted at publishers, Enterprise brands and digital agencies
+which seek to offer a seamless experience to their website visitors, 100% of
+the time. 
+
+Recently, we have developed an innovative component to our core
+product—WordPress Business Intelligence. Users can now get real time,
+actionable data in a comprehensive dashboard to support a short
+issue-to-deployment process and continuous improvement of their sites.
+
+We support the seamless delivery of up to 2 billion pageviews per month, on a
+fleet of 100 machines entirely dedicated to managed WordPress hosting for
+demanding customers. 
+
+We’re currently on our mission to bring the best experience to WordPress
+publishers around the world. In this journey, Kubernetes facilitates our route
+to an upcoming standard in high availability WordPress hosting infrastructure.
+
+## What was your pre-Prometheus monitoring experience?
```

Comment:
"pre-Prometheus" is awkward. How about "previous"?

Reply:
That's what we've been using with the rest of them